docs/manual: Runtime chapter has a section that has broken indentation

### Steps to reproduce
1. Open the ocaml manual Chapter 15 runtime system https://ocaml.org/manual/5.4/runtime.html
2. Check the section that describes the OCAMLRUNPARAM environment variable

### Expected Behavior
The section describes the environment variables options `b` to `W`

### Actual Behavior
The section breaks indentation around `n` and  `o`

### Other details
I tried building the manual locally and was able to edit the runtime.etex file; You can view the commit [here](https://github.com/curche/ocaml/commit/b4fa43ce5d15084f5529ca59d4da1a0636465b63). 
I was able to generate the manual using `make web` in `manual/src` folder and the rendered output seems to have proper indentation. If this is sufficient, I can rebase my changes and send in a PR. In case there's other feedback or alternative methods, I'll be happy to help

<!--
Welcome to OCaml's Issue tracker!

OCaml's developers use this tracker for bugs and feature requests only, rather
than user support.

If you have questions about *using* OCaml, please ask at
https://discuss.ocaml.org (more people read Discuss than this tracker, and
you'll get confirmation of whether you've really found a bug or need a new
feature).

If your error came from the OCaml package manager, opam, (messages beginning
`[ERROR] The compilation of ...`), please start at
https://github.com/ocaml/opam-repository/issues/new.

Some libraries and tools which used to be part of OCaml are now maintained
separately. Please post questions about Graphics, Num, camlp4, LablTk, CamlDBM
or OCamlTopWin on Discuss or on their respective issue trackers (see [README.adoc](https://github.com/ocaml/ocaml/blob/trunk/README.adoc#separately-maintained-components)
for a full list).
-->

--- a/manual/src/cmds/runtime.etex
+++ b/manual/src/cmds/runtime.etex
@@ -159,9 +159,6 @@ The following environment variables are also consulted:
         the rest is directly counted against "custom_major_ratio".
         Default: 8192 bytes. Note:
         this only applies to values allocated with "caml_alloc_custom_mem".
-        \end{options}
-        The multiplier is "k", "M", or "G", for multiplication by $2^{10}$,
-        $2^{20}$, and $2^{30}$ respectively.
   \item[o] ("space_overhead")  The main major GC speed setting.
         See the Gc module documentation for details.
   \item[p] (parser trace) Turn on debugging support for
@@ -200,7 +197,10 @@ The following environment variables are also consulted:
     the completion of a major GC cycle
   \item[W] Print runtime warnings to stderr (such as Channel opened on file
     dies without being closed, unflushed data, etc.)
+  \end{options}
 
+  The multiplier is "k", "M", or "G", for multiplication by $2^{10}$,
+  $2^{20}$, and $2^{30}$ respectively.
   If the option letter is not recognized, the whole parameter is ignored;
   if the equal sign or the number is missing, the value is taken as 1;
   if the multiplier is not recognized, it is ignored.

